helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -19,44 +17,18 @@ ingressController:
   legacy: false
   replicas: 3
 
-# configmap contains settings that can be overridden with a custom values
-# configmap.
```

Comment:
All of the settings in values.yaml can be overriden with a custom values configmap.

`.Values.configmap` is now purely for nginx configmap config options. Everything else that doesn't actually belong here moved under different configuration key. Settings which were under configmap just to support overriding but weren't actually overriding upstream defaults were removed since that is no longer needed. Similarly, configmap entries which have same value as upstream default, so not overriding it, were removed. Existing actual overrides were kept. Settings no longer supported upstream were cleaned up.